packages/docs/docs/api/fhir/operations/conceptmap-translate.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 9
+---
+
+# ConceptMap Translate
+
+Medplum implements the [`ConceptMap/$translate`][translate-operation] operation, which allows mapping a coded value from
+
+[translate-operation]: http://hl7.org/fhir/R4/conceptmap-operation-translate.html
+
+## Invoke the `$translate` operation
+
+```
+[base]/ConceptMap/$translate
+[base]/ConceptMap/[id]/$translate
+```
+
```

Comment:
Add a table with the expected parameters, similar to here: 

https://www.medplum.com/docs/api/fhir/operations/valueset-expand